src/v2/Apps/Artwork/Components/ArtworkZendesk.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ZendeskWrapper } from "v2/Components/ZendeskWrapper"
+import { isExceededZendeskThreshold } from "v2/Utils/isExceededZendeskThreshold"
+import { ArtworkZendesk_artwork } from "v2/__generated__/ArtworkZendesk_artwork.graphql"
+
+interface ArtworkZendeskProps {
+  artwork: ArtworkZendesk_artwork
+}
+
+const ArtworkZendesk: FC<ArtworkZendeskProps> = ({ artwork }) => {
+  const {
+    isAcquireable,
+    isInquireable,
+    isOfferable,
+    listPrice,
+    isInAuction,
+  } = artwork
+
+  // Don't display on inquiry artworks
+  if (isInquireable && !isAcquireable && !isOfferable) {
+    return null
+  }
+
+  const price =
+    listPrice?.__typename === "Money"
+      ? listPrice
+      : listPrice?.__typename === "PriceRange"
+      ? listPrice.maxPrice
+      : null
```

Comment:
**nitpick [non-blocking]:** avoid nested ternary

Could this be a map instead of a nested ternary? I'm thinking about something like:

```ts
const map = {
  Money: listPrice,
  PriceRange: listPrice.maxPrice
}
const price = typeToPriceMap[listPrice?.__typename]
```

What do you think?

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
Yeah that's nicer. Forgot to replace this, I just copied this logic from the old component.

Reply:
Replaced with a switch statement for type safety.